version.json

```diff
@@ -2,6 +2,6 @@
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
   "version": "0.1-alpha",
   "publicReleaseRefSpec": [
-    "^refs/heads/master$"
+    "^refs/tags/v\\d+\\.\\d+\\.\\d+"
```

Comment:
Aren't our tags are alphanumeric?
![image](https://user-images.githubusercontent.com/22460039/93925107-7b460080-fcca-11ea-8273-713f677fcf8b.png)

Reply:
There's no `$` at the end of the regex, so it's just looking for strings starting vX.Y.X

Reply:
Yeah good pt.